cnab-config/castai-agent/createUIDefinition.json

```diff
@@ -0,0 +1,211 @@
+{
```

Comment:
what is this used for?

Reply:
https://learn.microsoft.com/en-us/azure/azure-resource-manager/managed-applications/create-uidefinition-overview
This is the config which controls  Azure K8s extension creation UI page.